### ISSUE TITLE
Migration error on example answers

### ISSUE THREAD

**Issue by sjDCC** [DMPRoadmap/roadmap#270]
Creating a plan on DMPonline test has shown more data migration issues.

Example answers are showing up on a plan I have created for the AHRC, based on being a user at the University of Edinburgh. However the UoE has not customised the AHRC template. The example answer being displayed comes from the University of St Andrew's. 
![example-answer-migration-error](https://cloud.githubusercontent.com/assets/6169940/25387064/67c2903a-29c1-11e7-92d5-d0c2a1e4d550.PNG)

Looking at the live DMPonline data, 4 institutions have added examples to this one AHRC question (St Andrew's, York, Lancaster and Liverpool). What has happened to the other data?

The AHRC guidance has also been transposed into UoE guidance. A known issue reported in #263

**Comment by xsrust** [comment]
related to (and blocked by) #223, which will move example answers into the annotations table.  Once that is done, we can check to see if the problem is with the data and that the organisation associated with the guidance is wrong, or if the problem is with the display logic.

**Comment by xsrust** [comment]
Addressed in migrations:
https://github.com/DMPRoadmap/roadmap/commit/0c74f1ad63f0e330525b9c3f06b4c5d6029332fb
https://github.com/DMPRoadmap/roadmap/commit/ebc5ed836900e629c2506d91b9265920500c8a33

Data will need to be re-migrated in order to facilitate testing of this ticket

**Comment by sjDCC** [comment]
This issue is still occurring - see ticket #381 recorded today

**Comment by dsisu** [comment]
Trying to test AHRC template but can't select it. 
![ahrcliv](https://cloud.githubusercontent.com/assets/2138956/26553591/38708232-4484-11e7-9cf9-90b8004a1f5f.JPG)
See screenshot.

**Comment by sjDCC** [comment]
@dsisu I think this is because it's not a controlled entry from the list. Try typing the funder again and select AHRC once you've typed 'Arts...'

**Comment by dsisu** [comment]
That's what I did and it doesn't work. It also inserts &amp; instead of &. We should just use 'and' if html coding causes problems.

**Comment by vyruss** [comment]
@dsisu that is a separate issue with the new combobox which doesn't select names which contain ampersands.

**Comment by dsisu** [comment]
If it's a separate issue do you want me to raise a separate ticket? You seem to be aware of it.

**Comment by vyruss** [comment]
@dsisu we'll handle it under #124

**Comment by sjDCC** [comment]
We need it to work for now so have changed the ampersand to 'and' and it lets you create a plan for AHRC.

Should I change this on the current live site so the correct data gets pulled through?

**Comment by vyruss** [comment]
It's a workaround, but yes @sjDCC please do change it until we can tackle the dropdown.

**Comment by xsrust** [comment]
this is fixed on dmponline-test data:

![screen shot 2017-05-31 at 2 26 36 pm](https://cloud.githubusercontent.com/assets/7352616/26634075/59388ba6-460d-11e7-9f92-98282fea292b.png)

**Comment by sjDCC** [comment]
Tested for several orgs and is working. Closing out.
